org.gitlab.neva_blyad.yat.yml

```diff
@@ -0,0 +1,286 @@
+      - -DFULLY_STATIC:BOOL=OFF     # Build fully static executables
+      - -DSTRICT_ABI:BOOL=OFF       # Enforce strict ABI export in dynamic libraries
+      - -Dpkgcfg_lib_LIBSODIUM_sodium:FILEPATH=/app/lib/libsodium.so         # Path to a library. TODO: do cross-platform.
+      - -Dpkgcfg_lib_OPUS_opus:FILEPATH=/usr/lib/x86_64-linux-gnu/libopus.so # Path to a library. TODO: do cross-platform.
+      - -Dpkgcfg_lib_VPX_m:FILEPATH=/usr/lib/x86_64-linux-gnu/libm.so        # Path to a library. TODO: do cross-platform.
+      - -Dpkgcfg_lib_VPX_vpx:FILEPATH=/usr/lib/x86_64-linux-gnu/libvpx.so    # Path to a library. TODO: do cross-platform.
+    sources:
+      - type: git
+        url: 'https://github.com/toktok/c-toxcore'
+        tag: v0.2.18
+        commit: 3a5da3588f693ba17768a9a4cbd67d54d53114ac
+        x-checker-data:
+          - type: git
+            tag-pattern: '^v([\d.]+)$'
+    post-install:
+      - sed -i 's|-I${includedir}|-I${includedir} -I${includedir}/tox -I${includedir}/opus|' /app/lib/pkgconfig/toxcore.pc
+    cleanup:
+      - /include/tox/
+      - /lib/pkgconfig/toxcore.pc
+      - /lib/libtoxcore.a
+
+  #-------------------------------------------------------------------------------
+  # SCons.
+  #
+  # Note. Generated automatically using flatpak-builder-tools:
+  #     $ flatpak-pip-generator --build-only --runtime='org.freedesktop.Platform//23.08' scons
+  #-------------------------------------------------------------------------------
+  - python3-scons.json
+
+  #-------------------------------------------------------------------------------
+  # wxWidgets
+  #-------------------------------------------------------------------------------
+  - name: wxwidgets
+    buildsystem: autotools
+    config-opts:
+      - --enable-display
+      - --enable-geometry
+      - --enable-graphics_ctx
+      - --enable-mediactrl
+      - --enable-sound
+      - --enable-webview
+      - --with-expat=sys
+      - --with-gtk=3
+      - --with-libjpeg=sys
+      - --with-libpng=sys
+      - --with-libtiff=sys
+      #- --with-opengl
+      - --with-zlib
+    sources:
+      - type: archive
+        url: 'https://github.com/wxWidgets/wxWidgets/releases/download/v3.0.5.1/wxWidgets-3.0.5.1.tar.bz2'
+        sha1: 406ac736f61d88a3a866aa501e01e408a642c6e7
+    cleanup:
+      - /bin
+      - /include
+      - /lib/wx
+      - /share
+
+  #-------------------------------------------------------------------------------
+  # wxC.
+  #
+  # TODO: use specific module version instead of the last commit
+  #-------------------------------------------------------------------------------
+  - name: wxc
+    buildsystem: simple
+    sources:
+      - type: git
+        url: 'https://gitlab.com/neva_blyad/wxc'
+    build-options:
+      env:
+        - ARCH=x86_64-linux-gnu # TODO: do cross-platform
+    build-commands:
+      - sed -i 's|prefix=/usr|prefix=/app|' wxc.pc.unix
```

Comment:
you should already fix this upstream to honour the prefix. Most build system regenerate it when the prefix is set.